tests/store/model_registry/test_sqlalchemy_store.py

```diff
@@ -810,6 +810,19 @@ def test_search_registered_models(self):
         self.assertEqual(self._search_registered_models("name ILIKE '{}%'".format(prefix + 'RM4A')),
                          ([names[4]], None))
 
+    def test_parse_search_registered_models_order_by(self):
+        # test that "registered_models.name ASC" is returned by default
+        parsed = SqlAlchemyStore._parse_search_registered_models_order_by([])
+        self.assertEqual([str(x) for x in parsed], ['registered_models.name ASC'])
+
+        # test that when order_by contains a name field, the given name field is used
+        parsed = SqlAlchemyStore._parse_search_registered_models_order_by(['name DESC'])
+        self.assertEqual([str(x) for x in parsed], ['registered_models.name DESC'])
+
+        # test that an exception is thrown when order_by contains duplicate fields
+        with self.assertRaisesRegex(MlflowException, 'order_by contains duplicate fields:'):
+            SqlAlchemyStore._parse_search_registered_models_order_by(['name', 'name'])
```

Comment:
Can we also get a few tests here that have the same field `ASC` and `DESC` (but not `name`, since we test that above, implicitly)

Reply:
Added a few more test cases